src/notation/internal/notationuiactions.cpp

```diff
@@ -563,12 +563,18 @@ const UiActionList NotationUiActions::m_actions = {
              TranslatableString("action", "Measure properties…"),
              TranslatableString("action", "Measure properties…")
              ),
-    UiAction("add-remove-breaks",
+    UiAction("add-remove-breaks", // DEPRECATED
              mu::context::UiCtxProjectOpened,
              mu::context::CTX_NOTATION_OPENED,
              TranslatableString("action", "Add/remove s&ystem breaks…"),
              TranslatableString("action", "Add/remove system breaks…")
              ),
+    UiAction("measures-per-system",
```

Comment:
Maybe make it start with a verb, like `lock-measures-per-system`

Reply:
I'd prefer to leave it as is so it's worded in the same way as the UI dialog connected to it